app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,10 @@
   </head>
 
   <body>
+    <% if flash[:notice] %>
+      <div class="notice"><%= flash[:notice] %></div>
+    <% end %>
+
     <%= yield %>
   </body>
 </html>
```

Comment:
This file is in html, needs to be changed to haml

Reply:
Ugh. :P 
It's the auto generated layouts file. We'll have a look and see if it makes sense to change it.

Reply:
Fixed